tests/main/ubuntu-core-refresh/task.yaml

```diff
@@ -0,0 +1,65 @@
+summary: check that the ubuntu-core system is rebooted after the core snap is refreshed
+
+details: |
+    This test checks that when invoking a manual refresh/revert for core or core18 snaps,
+    it is triggered a reboot and the command would exit after the first phase of the installation
```

Comment:
Nitpick, "it is triggered a reboot" -> "a reboot is triggered"

Reply:
done

Reply:
done